src/Uno.UI/UI/Xaml/Controls/TextBox/MultilineTextBoxDelegate.iOS.cs

```diff
@@ -66,7 +66,15 @@ public override bool ShouldChangeText(UITextView textView, NSRange range, string
 					{
 						var newLength = bindableTextView.Text.Length + text.Length - range.Length;
 						return newLength <= textBox.MaxLength;
```

Comment:
Same comment here about the correctness of this return.

Reply:
Thanks Youssef. Fixed